src/main/java/org/openstreetmap/atlas/tags/SmoothnessTag.java

```diff
@@ -0,0 +1,45 @@
+package org.openstreetmap.atlas.tags;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM smoothness tag
+ *
+ * @author bbreithaupt
+ */
+@Tag(taginfo = "https://taginfo.openstreetmap.org/keys/smoothness", osm = "https://wiki.openstreetmap.org/wiki/Key:smoothness")
+public enum SmoothnessTag
+{
+    EXCELLENT,
+    GOOD,
+    INTERMEDIATE,
+    BAD,
+    VERY_BAD,
+    HORRIBLE,
+    VERY_HORRIBLE,
```

Comment:
Would love to see some mapillary street-level images comparing `horrible` to `very_horrible` 😂